dotcom-rendering/src/components/Branding.amp.tsx

```diff
@@ -20,6 +20,7 @@ const LinkStyle = (pillar: ArticleTheme) => css`
 
 const brandingStyle = (pillar: ArticleTheme) => css`
 	padding: 10px 0;
+	width: 160px;
```

Comment:
The source image is meant to be high-DPI / retina / @2x with a width of 280, so the base width should always be 140px.
```suggestion
	width: 140px;
```